test/jdk/java/lang/Math/WorstCaseTests.java

```diff
@@ -114,8 +114,8 @@ private static int testWorstExp() {
             {+0x1.A8EAD058BC6B8p3,      0x1.1D71965F516ADp19},
             {+0x1.1D5C2DAEBE367p4,      0x1.A8C02E974C314p25},
             {+0x1.C44CE0D716A1Ap4,      0x1.B890CA8637AE1p40},
-            {+0x4.0p8,                  Double.POSITIVE_INFINITY},
-            {+0x2.71p12,                Double.POSITIVE_INFINITY},
+            {+0x4.0p8,                  Double.POSITIVE_INFINITY}, //bug 8255368 gave 0
```

Comment:
This is not an appropriate test to update to cover this bug. This test is specifically probing at difficult cases in double arithmetic for the underlying mathematically function as opposed to flaws in a particular implementation.